Package:     capi4hylafax
Subject:     fix spelling errors
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   no
Last-Update: 2012-06-17

Fix some spelling errors. They give always warnings.
---

diff -urN s08/src/faxrecv/faxrecv.cpp s09/src/faxrecv/faxrecv.cpp
--- s08/src/faxrecv/faxrecv.cpp	2012-06-13 21:28:02.000000000 +0200
+++ s09/src/faxrecv/faxrecv.cpp	2012-06-17 16:30:24.038671951 +0200
@@ -112,7 +112,7 @@
     if (Controller) {
         WriteLog (LOG_ERR, "Try to listen for call on Controller %d, but got error code 0x%X.\n", Controller, Info);
     } else if (CapiInfoIsRegError (Info)) {
-        // Get a Register-Errror as Reason => CAPI not installed, terminate Programm
+        // Get a register error as reason => CAPI not installed, terminate program
         dprint ("\nCAPI not installed! ");
         WriteLog (LOG_ERR, "CAPI not installed, started or have no access rights on it!\n");
         FaxDevice->WriteStatus ("CAPI not installed, started or have no access rights on it!!");
@@ -160,7 +160,7 @@
     }
     if (GetFormat() == FaxFormat_Hylafax) {
         if (GetJobFile() == vFalse) {
-            WriteLog (LOG_ERR, "Can't create log file for incomming call!\n");
+            WriteLog (LOG_ERR, "Can't create log file for incoming call!\n");
             // TODO: better disconnect the call?
         }
         WriteLog (LOG_INFO, "SESSION BEGIN %09u %s\n", jobNr, FaxDevice->FaxNumber.GetPointer());
diff -urN s08/src/faxrecv/recvmain.cpp s09/src/faxrecv/recvmain.cpp
--- s08/src/faxrecv/recvmain.cpp	2012-06-13 21:28:02.000000000 +0200
+++ s09/src/faxrecv/recvmain.cpp	2012-06-17 16:31:17.971227045 +0200
@@ -490,13 +490,13 @@
         }
         switch (errflags) {
         case 0:
-            WriteLog (LOG_INFO, "Changed to effective user \"%s\" and group \"%s\" succesfully.\n", FaxUser.GetPointer(), FaxGroup.GetPointer());
+            WriteLog (LOG_INFO, "Changed to effective user \"%s\" and group \"%s\" successfully.\n", FaxUser.GetPointer(), FaxGroup.GetPointer());
             break;
         case 1:
-            WriteLog (LOG_INFO, "Changed to effective user \"%s\" succesfully.\n", FaxUser.GetPointer());
+            WriteLog (LOG_INFO, "Changed to effective user \"%s\" successfully.\n", FaxUser.GetPointer());
             break;
         case 2:
-            WriteLog (LOG_INFO, "Changed to effective group \"%s\" succesfully.\n", FaxGroup.GetPointer());
+            WriteLog (LOG_INFO, "Changed to effective group \"%s\" successfully.\n", FaxGroup.GetPointer());
             break;
         default:
             break;
diff -urN s08/src/linuxdep/PrctlDmp.cpp s09/src/linuxdep/PrctlDmp.cpp
--- s08/src/linuxdep/PrctlDmp.cpp	2006-08-15 11:13:30.000000000 +0200
+++ s09/src/linuxdep/PrctlDmp.cpp	2012-06-17 16:32:01.161381351 +0200
@@ -49,9 +49,9 @@
                 fprintf (stderr, "Error: PR_SET_DUMPABLE failed - %s (%d)\n", strerror(errno), errno);
             }
         } else if (prctl(PR_GET_DUMPABLE) != 0) {
-            //printf ("Info: programm will dump core if needed");
+            //printf ("Info: program will dump core if needed");
         } else {
-            fprintf (stderr, "Error: programm not dumpable after PR_SET_DUMPABLE :-(\n");
+            fprintf (stderr, "Error: program not dumpable after PR_SET_DUMPABLE :-(\n");
         }
     }
 }
